component/src/test/java/org/wso2/extension/siddhi/store/ignite/DefineTableTestCase.java

```diff
@@ -0,0 +1,276 @@
+        siddhiAppRuntime.start();
+        siddhiAppRuntime.shutdown();
+    }
+
+    @Test(description = "Testing table creation with invalid url ")
+    public void createTableWithInvalidUrlTest() {
+        log.info("createTableWithInvalidUrlTest");
+        SiddhiManager siddhiManager = new SiddhiManager();
+        String streams = "" +
+                "define stream StockStream (symbol string, price float, volume long); " +
+                "@Store(type=\"apacheignite\", url = \"" + "jdbc:ignite:thi://127.0.1.1/" + "\" ," +
+                "username=\"" + USERNAME + "\", password=\"" + PASSWORD
+                + "\")\n" +
+                "@PrimaryKey(\"symbol\")" +
+                "define table StockTable (symbol string, price float, volume long); ";
+        SiddhiAppRuntime siddhiAppRuntime = siddhiManager.createSiddhiAppRuntime(streams);
+        siddhiAppRuntime.start();
+        siddhiAppRuntime.shutdown();
+    }
+
+    @Test(description = "Testing table creation without defining value for username field ",
+            expectedExceptions = SiddhiAppCreationException.class)
+    public void createTableWithoutDefineUsernameTest() {
+        log.info("createTableWithoutDefineUsernameTest");
+        SiddhiManager siddhiManager = new SiddhiManager();
+        String streams = "" +
+                "define stream StockStream (symbol string, price float, volume long); " +
+                "@Store(type=\"apacheignite\", url = \"" + URL + "\" ," + "auth.enabled=\"true\"," +
+                "username=\"" + "\", password=\"" + PASSWORD
+                + "\")\n" +
+                "@PrimaryKey(\"symbol\")" +
+                "define table StockTable (symbol string, price float, volume long); ";
+        SiddhiAppRuntime siddhiAppRuntime = siddhiManager.createSiddhiAppRuntime(streams);
+        siddhiAppRuntime.start();
+        siddhiAppRuntime.shutdown();
+    }
+
+    @Test(description = "Testing table creation with incorrect username field ")
+    public void createTableWithIncorrectUsernameTest() {
+        log.info("createTableWithIncorrectUsernameTest");
+        SiddhiManager siddhiManager = new SiddhiManager();
+        String streams = "" +
+                "define stream StockStream (symbol string, price float, volume long); " +
+                "@Store(type=\"apacheignite\", url = \"" + URL + "\" ," + "auth.enabled=\"true\"," +
+                "username=\"" + "ignit" + "\", password=\"" + PASSWORD
+                + "\")\n" +
+                "@PrimaryKey(\"symbol\")" +
+                "define table StockTable (symbol string, price float, volume long); ";
+        SiddhiAppRuntime siddhiAppRuntime = siddhiManager.createSiddhiAppRuntime(streams);
+        siddhiAppRuntime.start();
+        siddhiAppRuntime.shutdown();
+    }
+
+    @Test(description = "Testing table creation with table.name ")
+    public void createTableTest() throws InterruptedException, SQLException {
```

Comment:
Better to add names like `testTableCreationWithName` like [this](https://github.com/siddhi-io/siddhi-io-http/blob/master/component/src/test/java/io/siddhi/extension/io/http/source/HttpBasicTestCase.java#L63)